include/rcpputils/filesystem_helper.hpp

```diff
@@ -269,7 +270,7 @@ RCPPUTILS_PUBLIC bool exists(const path & path_to_check);
  *
  * \return A path to a directory for storing temporary files and directories.
  */
-RCPPUTILS_PUBLIC path temp_directory_path();
+RCPPUTILS_PUBLIC std::filesystem::path temp_directory_path();
```

Comment:
I'm going to suggest that we just remove this API completely, as it already exists in [std::filesystem](https://en.cppreference.com/w/cpp/filesystem/temp_directory_path):

```suggestion
```